### DISCUSSION TITLE
Improve dark mode support

### DISCUSSION THREAD

**Discussion by talmo** [talmolab/sleap#1865] [Ideas]
![image](https://github.com/user-attachments/assets/19ecbc8c-05ee-4ba5-a9d9-0d7a8bc909b0)

I think we're hardcoding the font colors for the status bar, but we should use a system light/dark mode aware default like the rest of the UI components.

**Comment by talmo** [comment]
![image](https://github.com/user-attachments/assets/80a5f511-6234-4360-acb1-98376eb5da24)

Another context clue for tracing down where we have to change Qt styling -- notice how the videos table, but not the nodes, gets the font color set appropriately for dark mode.

**Comment by roomrys** [comment]
I ran into this bug on the multiview branch a while back ([a5b47](https://github.com/talmolab/sleap/pull/1283/commits/a5b47698b6dd119ed3cb8d787ce9ac69fa71ad35)) and realized it could also be the cause for the Skeletons table being colored black always. The problem is actually because we return black for the color of the item if there is no track index:
https://github.com/talmolab/sleap/blob/91d6c96dd77bc092d7835046245fb787b444523d/sleap/gui/color.py#L176-L177

and we are coloring by instances:
https://github.com/talmolab/sleap/blob/91d6c96dd77bc092d7835046245fb787b444523d/sleap/gui/color.py#L260

It seems like the intention [here](https://github.com/talmolab/sleap/commit/8c8c97846aafb6cb3f7e6b75a0a0c0dd9fb2bb3c) was to color the Skeletons table the same color as the skeleton:
https://github.com/talmolab/sleap/blob/91d6c96dd77bc092d7835046245fb787b444523d/sleap/gui/dataviews.py#L416-L421

I am creating a feature branch and fix right now. Just wanted to update so that no one else goes down a rabbit hole.
- #1952

**Comment by eberrigan** [comment]
Bottom-bar still needs to be dependent on UI mode through QT.

**Comment by 7174Andy** [comment]
Bottom-bar fixed in https://github.com/talmolab/sleap/pull/2063

**Comment by roomrys** [comment]
Long awaited, but finally integrated. This issue has been fixed in the SLEAP 1.4.1 release!
